dtu-hpc-python3/setup-python3.sh

```diff
@@ -246,6 +246,10 @@ export PYTHON_BIN_PATH=`which python3`
 export TF_NEED_GCP=0
 export TF_NEED_HDFS=0
 export TF_NEED_CUDA=1
+export TF_NEED_JEMALLOC=0
+export TF_NEED_GCP=0
```

Comment:
This is true, but if `TF_NEED_OPENCL` isn't set `.configure` will prompt the user (the `yes ""` command) for input. Setting `TF_NEED_OPENCL=0` skips the prompt and disables OpenCL.